src/stratis_cli/_error_reporting.py

```diff
@@ -145,6 +145,16 @@ def interpret_errors(errors):
             and error.get_dbus_name() == "org.freedesktop.DBus.Error.NameHasNoOwner"
         ):
             return "Most likely stratis is unable to connect to the stratisd D-Bus service."
+        if (
```

Comment:
Please go ahead and add the three line FIXME, like with the other parts of this method that we are not testing.